UnityProject/Assets/Scripts/Weapons/Projectiles/Behaviours/ProjectileDamageLivingHealth.cs

```diff
@@ -28,11 +30,13 @@ public bool OnHit(RaycastHit2D hit)
 
 		private bool TryDamage(RaycastHit2D hit)
 		{
+			var newDamage = damageData.Damage;
```

Comment:
since you're not doing any further calculations, just use ``damageData.Damage``.